config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -1,4 +1,34 @@
 presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-gci-gce-ipvs
+    always_run: false
+    optional: true
```

Comment:
intentionally left out `run_if_changed` for now, will reevaluate based on how stable this is

Reply:
should we skip_report until we make it "green"?

> If you only want to run tests when specific files are touched, you can use run_if_changed. A useful pattern when adding new jobs is to start with always_run set to false and skip_report set to true. Test it out a few times by manually triggering, then switch always_run to true. Watch for a couple days, then switch skip_report to false.